.pytest_cache/README.md

```diff
@@ -0,0 +1,8 @@
+# pytest cache directory #
```

Comment:
this directory is not supposed to be comitted

Reply:
> can you please rebase on latest master?

yes, doing that.